### DISCUSSION TITLE
Adding "Remove block" prompt

### DISCUSSION THREAD

**Discussion by seba84** [WordPress/gutenberg#35137] [Q&A]
Is it possible to add a prompt/popup message that will ask you "are sure that you want to remove that block?" and then offer yes or no?
![25](https://user-images.githubusercontent.com/26061521/134851502-7e78e9ca-b784-4598-9013-bdcc8e1a9f96.JPG)
.

**Comment by jasmussen** [comment]
Hi, thanks for the question. Are you sure that's necessary? There's both undo and version history if you accidentally remove a block.

**Reply by shaunandrews** [reply]
For most blocks, I agree with @jasmussen: Considering the ability to undo, along with revisions, I'm not sure a confirmation is necessary for removing a block.

However, I could see a need for a confirmation when removing _some_ blocks, like a Template Part block.
